cvat/apps/quality_control/quality_reports.py

```diff
@@ -2015,9 +2030,8 @@ def _make_zero_confusion_matrix(self) -> Tuple[List[str], np.ndarray, Dict[int,
 
         return label_names, confusion_matrix, label_id_idx_map
 
-    @classmethod
-    def _generate_annotations_summary(
-        cls, confusion_matrix: np.ndarray, confusion_matrix_labels: List[str]
+    def _compute_annotation_summary(
```

Comment:
```suggestion
    def _compute_annotations_summary(
```

This would be consistent with the result type name.